src/index.ts

```diff
@@ -62,17 +72,23 @@ const useDimensions = <T extends HTMLElement>({
   });
   const prevSizeRef = useRef<{ width?: number; height?: number }>({});
   const prevBreakpointRef = useRef<string>();
-  const observerRef = useRef<ResizeObserver | null>(null);
+  const observerRef = useRef<any | null>(null);
```

Comment:
@runar-rkmedia Why we need to change this type?

Reply:
sorry, guess I missed that one during developmen. I'll revert that change.